test/box-luatest/gh_10622_net_box_trigger_crash_on_self_delete_test.lua

```diff
@@ -0,0 +1,68 @@
+local server = require('luatest.server')
+local t = require('luatest')
+
+local g = t.group()
+
+g.before_all(function()
+    g.server = server:new({alias = 'master'})
+    g.server:start()
+end)
+
+g.after_all(function()
+    g.server:stop()
+end)
+
+-- Reproducer from the issue
+g.test_net_box_trigger_crash_on_self_delete = function()
+    local server2 = server:new({alias = 'master'})
```

Comment:
Why not use `server` here?

Reply:
Fixed, use server here so that it connects to itself.